OtripleS.Portal.Web/Services/StudentViews/StudentViewService.cs

```diff
@@ -75,5 +86,18 @@ private Student MapToStudent(StudentView studentView)
                 UpdatedDate = currentDateTime
             };
         }
+        
+        private static Func<Student, StudentView> AsStudentView = delegate (Student student)
```

Comment:
You don't really need to use `delegate` in here you can just do the fat-arrow example.

Reply:
Do like TeacherViewService one?

Reply:
Exactly.

```csharp
    private static Func<Teacher, TeacherView> AsTeacherView =>
            teacher => new TeacherView
            {
                EmployeeNumber = teacher.EmployeeNumber,
                FirstName = teacher.FirstName,
                MiddleName = teacher.MiddleName,
                LastName = teacher.LastName,
                Gender = (TeacherGenderView)teacher.Gender,
                Status = (TeacherStatusView)teacher.Status,
            };

```